test/integ/axpy/src/main.cpp

```diff
@@ -97,7 +98,7 @@ struct VectorAddKernelTester
         using Val = float;
 
         // Create the kernel function object.
-        VectorAddKernel kernel;
+        AxpyKernel kernel;
 
         // Get the host device.
         auto devHost(alpaka::dev::cpu::getDev());
```

Comment:
Should we use getDevByIdx(0) here ? I know that it is the same, but it is more explicit (we did the same in the examples).